packages/website/src/components/hooks/useDelayToggle.ts

```diff
@@ -0,0 +1,26 @@
+import { useRef, useCallback, useState } from 'react';
+
+export default function useDelayToggle<T>(
```

Comment:
FYI - the name for this pattern is called "debounce"
```suggestion
export default function useDebouncedToggle<T>(
```

Reply:
awesome, I'm bad with naming stuff